src/test/java/de/zalando/aruha/nakadi/utils/TestUtils.java

```diff
@@ -89,6 +89,10 @@ public static String resourceAsString(final String resourceName, final Class cla
         return IOUtils.toString(clazz.getResourceAsStream(resourceName));
     }
 
+    public static String getStringFromFile(final String resourceName) throws IOException {
```

Comment:
I remember we do it in several places in code already, so your method will be a god use there.
Can you please search for `Resources.toString` and replace what you found with this new `getStringFromFile` method you created.